### ISSUE TITLE
zsh:permission denied:drush on MacOS Ventura on Apple Silicon (M2)

### ISSUE THREAD

**Issue by SebastienGicquel** [drush-ops/drush-launcher#104]
I installed drush launcher under MacOs Ventura on a Mac M2 by following the procedure.

After having done
`sudo mv drush.phar /usr/local/bin/drush`

When I do `drush st` for example, I have this error
`zsh:permission denied:drush`

I changed the permissions but I still get the error. Can you give me some advice ?

Should we put it somewhere else?

Thanks for your help

**Comment by leymannx** [comment]
Cross posted on Drupal Answers: https://drupal.stackexchange.com/q/315511/15055. The README already contains the step on how you should change the permission correctly.

**Comment by SebastienGicquel** [comment]
I thought I had followed the procedure correctly, but I made a mistake. Since there was no folder in `user/local`, I created the `bin` and `drush` folder which was an error. We just have to have this path `/usr/local/bin/`and run the command `sudo mv drush.phar /usr/local/bin/drush` that will copy `drush.phar` in ` /usr/local/bin/` and rename `drush.phar` in `drush`.

Thanks a lot for your help.
